src/components/PageBlocks/FileConfigurator.vue

```diff
@@ -76,7 +222,13 @@ export default {
     },
 
     enableFileNameHiding () {
```

Comment:
Do we need these two exact same computed properties? 
Do the same as in field configurator